x/leverage/gov_handler_test.go

```diff
@@ -30,30 +30,93 @@ func TestUpdateRegistryProposalHandler(t *testing.T) {
+			LiquidationIncentive: sdk.MustNewDecFromStr("0.1"),
+			EnableLend:           true,
+			EnableBorrow:         true,
+			Blacklist:            false,
+		}))
+		require.NoError(t, k.SetRegisteredToken(ctx, types.Token{
+			BaseDenom:            "uatom",
+			SymbolDenom:          "ATOM",
+			Exponent:             6,
+			ReserveFactor:        sdk.MustNewDecFromStr("0.20"),
+			CollateralWeight:     sdk.MustNewDecFromStr("0.25"),
+			LiquidationThreshold: sdk.MustNewDecFromStr("0.25"),
+			BaseBorrowRate:       sdk.MustNewDecFromStr("0.02"),
+			KinkBorrowRate:       sdk.MustNewDecFromStr("0.22"),
+			MaxBorrowRate:        sdk.MustNewDecFromStr("1.52"),
+			KinkUtilizationRate:  sdk.MustNewDecFromStr("0.8"),
+			LiquidationIncentive: sdk.MustNewDecFromStr("0.1"),
+			EnableLend:           true,
+			EnableBorrow:         true,
+			Blacklist:            false,
+		}))
 
 		p := &types.UpdateRegistryProposal{
 			Title:       "test",
 			Description: "test",
 			Registry: []types.Token{
-				{BaseDenom: "uumee"},
-				{BaseDenom: "uatom", BaseBorrowRate: sdk.MustNewDecFromStr("0.02")},
+				{
+					BaseDenom:            umeeapp.BondDenom,
+					SymbolDenom:          "UMEE",
+					Exponent:             6,
+					ReserveFactor:        sdk.MustNewDecFromStr("0.20"),
+					CollateralWeight:     sdk.MustNewDecFromStr("0.25"),
+					LiquidationThreshold: sdk.MustNewDecFromStr("0.25"),
+					BaseBorrowRate:       sdk.MustNewDecFromStr("0.02"),
+					KinkBorrowRate:       sdk.MustNewDecFromStr("0.22"),
+					MaxBorrowRate:        sdk.MustNewDecFromStr("1.52"),
+					KinkUtilizationRate:  sdk.MustNewDecFromStr("0.8"),
+					LiquidationIncentive: sdk.MustNewDecFromStr("0.1"),
+					EnableLend:           true,
+					EnableBorrow:         true,
+					Blacklist:            false,
+				},
+				{
+					BaseDenom:            "uosmo",
+					SymbolDenom:          "OSMO",
+					Exponent:             6,
+					ReserveFactor:        sdk.MustNewDecFromStr("0.20"),
+					CollateralWeight:     sdk.MustNewDecFromStr("0.25"),
+					LiquidationThreshold: sdk.MustNewDecFromStr("0.25"),
+					BaseBorrowRate:       sdk.MustNewDecFromStr("0.02"),
+					KinkBorrowRate:       sdk.MustNewDecFromStr("0.22"),
+					MaxBorrowRate:        sdk.MustNewDecFromStr("1.52"),
+					KinkUtilizationRate:  sdk.MustNewDecFromStr("0.8"),
+					LiquidationIncentive: sdk.MustNewDecFromStr("0.1"),
+					EnableLend:           true,
+					EnableBorrow:         true,
+					Blacklist:            false,
+				},
 			},
 		}
 		require.NoError(t, h(ctx, p))
 
 		tokens := k.GetAllRegisteredTokens(ctx)
-		require.Len(t, tokens, 2)
+		require.Len(t, tokens, 3)
 
-		_, err := k.GetRegisteredToken(ctx, "uosmo")
-		require.Error(t, err)
-
-		_, err = k.GetRegisteredToken(ctx, "uumee")
+		// ensure that new tokens was added
+		_, err := k.GetRegisteredToken(ctx, "uumee")
 		require.NoError(t, err)
 
+		// ensure that unaffected token was not deleted by the gov proposal
```

Comment:
```suggestion
		// ensure that only the specified token was deleted
```

Reply:
no tokens were deleted - I'll clarify comment in a different way